microsoft-edge/dualengine/index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Introduction to Microsoft DualEngine
+description: Host web content in your Win32 apps with the Microsoft DualEngine interface.
+author: MSEdgeTeam
+ms.author: edgededev
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.localizationpriority: high
+ms.technology: windows-integration
+ms.date: 11/06/2023
+---
+# Introduction to Microsoft DualEngine API
+
+The Microsoft DualEngine API allows you to embed and control a Internet Explorer within your app.

```

Comment:
```suggestion
The Microsoft DualEngine API allows you to embed and control an Internet Explorer instance within your app.
```
alts:
* The Microsoft DualEngine API allows you to embed and control an Internet Explorer instance within your app.
* The Microsoft DualEngine API allows you to embed and control an Internet Explorer control within your app.
* The Microsoft DualEngine API allows you to embed and control an Internet Explorer window within your app.
* The Microsoft DualEngine API allows you to embed and control an Internet Explorer pane within your app.

Reply:
update source file too?  Updated to:
The Microsoft DualEngine API enables you to host and control an Internet Explorer instance within your app.

i changed to that wording in the gen'd .md file today, you might want to apply that change to the actual source file @AndrewLakeMSFT

Reply:
The .md is a static, not gen'd file.  No longer "index.md"; now "intro.md".  

Latest Files Changed tab includes the word 'instance':

Use the Microsoft DualEngine API to embed and control an Internet Explorer instance within your app.  The DualEngine API provides a browsing experience that's equivalent to a normal Internet Explorer tab, to provide increased compatibility with legacy websites.

The DualEngine API differs from similar APIs like the Web Browser Control, and requires distinct development work.